scripts/upgrade_pyyaml.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2021 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import os
```

Comment:
THANK YOU for writing this in Python instead of bash, even if it's not as concise.